app/src/main/java/ch/hikemate/app/ui/components/HikeCard.kt

```diff
@@ -64,7 +64,7 @@ data class HikeCardStyleProperties(
 fun HikeCard(
     title: String,
     modifier: Modifier = Modifier,
-    elevationData: List<Double> = emptyList(),
+    elevationData: List<Double>?,
```

Comment:
Suggestion: I would recommend you pass the hiking routes viewModel to this composable instead of the data directly and add a `getAltitudeDifference` function to the viewModel. This makes this feature available to other screens as well, such as for the Hike Details screen, so we avoid having to reimplement this different calculation multiple times.

Reply:
That's a good suggestion. However I did that in the beginning which wasn't working.
It wasn't as the `HikeCard` would not reload because it's inside a `LazyColum` `item`. That's why I used a `map` for storing the elevation datas.

Reply:
I'm not sure I understand. The HikeCard should reload once it gets re-displayed in the column, or?

Reply:
The issue is that an item inside a `LazyColum` will be kept in memory in order to not redraw it unless we remove it from screen and put it back.

Reply:
I see, thanks for clarifying.